tasks/cleanup-workspace/cleanup-workspace.yaml

```diff
@@ -15,6 +15,10 @@ spec:
     - name: subdirectory
       type: string
       description: The directory to remove within the workspace
+    - name: delay
+      type: string
+      default: 60s
```

Comment:
Optional: I would prefer using just "60" without the "s". For example, the `sleep` version available in macOS does not support the unit suffix. Which of course shouldn't be an issue, as you'll never really run the task in macOS directly.

Reply:
removed the 's'

Reply:
Cool, thanks.